Marlin/Marlin_main.cpp

```diff
@@ -1906,13 +1907,16 @@ static void clean_up_after_endstop_or_probe_move() {
 
     if (!endstops.z_probe_enabled) return;
 
+    // Make room for the probe
+    do_probe_raise(Z_RAISE_BETWEEN_PROBINGS);
```

Comment:
I'm not sure about this raise, which assumes the probe is "not really" stowing. I assume if "stow_probe" is being called, then the user really wants a raise to the `AFTER` value, even in the case of `G29 E`. In fact, it's arguable that for `G29 E` it's natural to assume that every deploy/stow will use the `BEFORE`/`AFTER` values, and we should meet that expectation. What do you think?